common/changes/@microsoft/rush/ianc-tar-logging_2021-04-19-19-34.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Print a log file to \"<project-root>/.rush/build-cache-tar.log\" when the native \"tar\" is invoked for debugging purposes.",
```

The grammar is ambiguous.
```suggestion
      "comment": "Print diagnostic information to a log file \"<project-root>/.rush/build-cache-tar.log\" when the native \"tar\" is invoked.",
```